src/frontend/app/features/endpoints/create-endpoint/create-endpoint-cf-step-1/create-endpoint-cf-step-1.component.html

```diff
@@ -1,14 +1,26 @@
+<h1>Register new Endpoint</h1>
```

Comment:
To be consistent with other pages, I think this should replace "Endpoints" in the blue header bar.